src/hooks/token.ts

```diff
@@ -16,6 +16,15 @@ export const networkMap = new Map([
   [100, "xdai"],
   [137, "polygon"],
   [56, "bsc"],
+  [30, "rsk"],
+  [31, "trsk"],
+]);
+
+const customNativeTokens = new Map([
+  [56, "BNB"],
+  [100, "xDai"],
+  [30, "RBTC"],
+  [31, "tRBTC"],
 ]);
```

Comment:
I like this! Might want to include all supported networks though. In order to keep all this in a single location I would suggest including these values directly in `networkMap` (for easier maintenance).